week-3-practise/2-filter-array.js

```diff
@@ -0,0 +1,36 @@
+/*
+Mix and ranking each team using .filter() function.
+*/
+
+var championsOrder = [[0, 4], [1, 2], [2, 3], null, [1], [3, 1], [4, 0]];
+
+var premierLeagueTeams = championsOrder.filter(indexPair => {
+  return indexPair !== null && indexPair.length;
```

Comment:
`indexPair.length` doesn't return a boolean, it returns a number, I think this needs a fix :)

Reply:
You are right. I made the correction.